### PR TITLE
Add examples to iotics-identity-rs- EI-794

### PR DESCRIPTION
**Pull request by mihai-dinculescu**

### CODE REVIEW

File: ffi/rust/examples/create_twin_did.rs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+/// Create a Twin DID
+/// Prerequisites: Create an User, an Agent and a delegation from the User to the Agent
+/// See the `create_user_and_agent` example
+fn main() {
+    // set up all the variables needed
+    let resolver_address = "https://did.stg.iotics.com";
+
+    let user_did = "did:iotics:iotNgaEk7erdRNZfuPczxFXs8M7LuYE8JVDk";
+
+    let agent_did = "did:iotics:iotKEeCnbYs9RM9DoPM9mdCK1At3qm7xvmwP";
```
Comment (Elleppi) [inline ffi/rust/examples/create_twin_did.rs:12]: Did you leave these values of `user_did` and `agent_did` on purpose ?

File: ffi/rust/examples/create_twin_did.rs [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+    // create the config object
+    let config = iotics_identity::Config {
+        resolver_address: resolver_address.to_string(),
+        user_did: user_did.to_string(),
+        agent_did: agent_did.to_string(),
+        agent_key_name: agent_key_name.to_string(),
+        agent_name: agent_name.to_string(),
+        agent_secret: agent_seed.to_string(),
+        token_duration: 600,
```
Comment (Elleppi) [inline ffi/rust/examples/create_twin_did.rs:28]: Pedantic: for the sake of these examples the token duration can be set to 1 second. However we usually tend to set it to high values so we don't have any sort of problems with it. Nevertheless if these are examples that can be potentially shown to and used by customers I think advertising high values of the token duration might lead to misunderstanding of its actual meaning

File: ffi/rust/examples/create_twin_did.rs (hunk 1 above)
Comment (mihai-dinculescu) [reply ffi/rust/examples/create_twin_did.rs:12]: Yes. They are used for creating the auth token.

File: ffi/rust/examples/create_twin_did.rs (hunk 2 above)
Comment (mihai-dinculescu) [reply ffi/rust/examples/create_twin_did.rs:28]: This value represents 10 minutes. Afaik it's a normal value for auth tokens. Best practices usually recommend 10-15 minutes. E.g. flask jwt extended uses 15 minutes https://flask-jwt-extended.readthedocs.io/en/stable/options/#JWT_ACCESS_TOKEN_EXPIRES

**Pull request merged**